setup.py

```diff
@@ -1,20 +1,21 @@
 from setuptools import setup, find_packages
 
 setup(
-    name = "destream",
-    version = "5.0.1",
-    author = "Cecile Tonglet",
-    author_email = "cecile.tonglet@cecton.com",
-    description = ("A simple module to decompress streams compressed multiple "
-                   "times"),
-    license = "GPLv2",
-    keywords = "stream file decompress zip zstd",
-    url = "https://github.com/cecton/destream",
-    packages = find_packages(),
-    scripts = ['scripts/destream'],
-    install_requires = ['python-magic>=0.4.12'],
+    name="destream",
+    version="5.0.1",
+    author="Cecile Tonglet",
+    author_email="cecile.tonglet@cecton.com",
```

Comment:
Pretty sure the author field is optional. I think we should remove it as it doesn't reflect all the contributors anyway. (It's also better for me as people won't try to reach me for support because of this)

Reply:
You are the original author. We can put Javier and me into the maintainer field though. Actually I have very little experience with what should be maintained which way on PyPI.